staging/src/github.com/openshift/template-service-broker/apis/template/constants.go

```diff
@@ -0,0 +1,50 @@
+package template
+
+// annotation keys
```

Comment:
can we put a huge WARNING here?

Reply:
Yeah, these are nasty, but we need to get something in.